IDEOorg/steps
# #161240710 Write tests for attemptClientLogin method

#### What does this PR do?
- This PR adds tests for attemptClientLogin function in `apps/admin/src/routes/index.tsx`
#### Description of Task to be completed?
- Export named Routes component
- Add tests for attemptClientLogin in index.test.tsx
#### How should this be manually tested?
- Run `npm test`
- Observe the test in `apps/admin/src/routes/index.test.tsx` passing
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#161240710](https://www.pivotaltracker.com/story/show/161240710)
#### Screenshots (if appropriate)
- None
#### Questions:
- N/A
